src/services/user.service.js

```diff
@@ -0,0 +1,47 @@
+const ApiError = require('../exeptions/api.error.js');
+const { User } = require('../models/user.js');
+const { v4: uuidv4 } = require('uuid');
+const emailService = require('../services/email.service');
+
+function getAllActivated() {
+  return User.findAll({
+    where: {
+      activationToken: null,
+    },
+  });
+}
+
+function normalize({ id, email }) {
+  return { id, email };
+}
+
+function findByEmail(email) {
+  return User.findOne({ where: { email } });
+}
+
+async function register(name, email, password) {
+  const activationToken = uuidv4();
+
+  const existUser = await findByEmail(email);
+
+  if (existUser) {
+    throw ApiError.badRequest('User already exist', {
+      email: 'User already exist',
+    });
```

Comment:
Ensure that the error handling here is consistent with the rest of the application. Consider using `next(ApiError.badRequest(...))` if this function is used in a context where `next` is available.